admin/includes/javascript_loader.php

```diff
@@ -31,6 +31,24 @@
 </script>
 <?php if (file_exists($jsFile = 'includes/javascript/' . basename($PHP_SELF, '.php') . '.js')) { ?>
   <script src="<?php echo $jsFile; ?>"></script>
+  <?php
+}
+if (file_exists($jsFile = 'includes/javascript/' . basename($PHP_SELF, '.php') . '.php')) {
+  require 'includes/javascript/' . basename($PHP_SELF, '.php') . '.php';
+}
+$template = new template_func();
```

Comment:
Since `javascript_loader.php` is called directly from `admin_html_head.php` which is already using `$template`, do we need to re-instantiate it again here?

Reply:
no